ils_middleware/tasks/alma/post_bfinstance.py

```diff
@@ -108,5 +114,7 @@ def putInstanceToAlma(
     put_mms_id = xml_response.xpath("//mms_id/text()")
     if put_update_status == 200:
```

Comment:
One suggestion would be to use a pattern match here instead of an `elif`. i.e.

```python
match put_update_status:

    case 200:
        task_instance.xcom_push(key=instance_uri, value=put_mms_id)

    case 500:
        raise Exception(f"Internal server error from Alma API: {put_update_status}")

    case _:
        raise Exception(f"Unexpected status code from Alma API: {put_update_status}")
```
This would make it easier to handle other PUT statuses in the future.